A14-channelz.md

```diff
@@ -71,23 +75,34 @@ The data representation of each ref:
 ![message ref structure][message ref structure]
 
 
-## Channels
+## Channels and Subchannels
 
-Channels are hierarchical organized into a tree structure.  A channel may have
-any number of "sub" channels.  Each subchannel is itself a channel and can also
-have subchannels.  Each channel may also have any number of sockets.   However,
-a given subchannel cannot have heterogeneous children.  That is, a channel may
-have subchannels or sockets, but not both.
+Channels and Subchannels are hierarchically organized into a DAG structure.  A
+channel may have any number of channels and subchannels.  Each subchannel can
+also have channels and subchannels.  Each channel and subchannel may also have
+any number of sockets.   However, a given channel or subchannel cannot have
+heterogeneous children.  That is, a channel or subchannel may have
+channels and subchannels, or  have sockets, but not both.
+
+A subchannel represents a channel that is load balanced over.  When a channel
+has both subchannels and channels, the channels are not delegated to by the 
+superchannel.  Channels and Subchannels will be used interchangeably unless
```

Comment:
To avoid confusion and make the spec clearer, I think it would be better to remove this last sentence.  Instead, we should look at every place in this document where we use the term "channel" and decide whether it applies to both channels and subchannels; if both, we should change it to say "channel or subchannel".

Reply:
Done.  There are a LOT of places, so I tried to unify them back together as "descendent channels".  I think I got each place.